src/components/Feeds/AllyFeed.js

```diff
@@ -15,23 +49,91 @@ const AllyFeed = data => {
     >
       {allAllies.length > 0 ? (
         <SimpleGrid
-          columns={[null, null, 2, 4]}
+          columns={[null, 1, 2, 4]}
           spacing={theme.spacing.med}
           paddingBottom={theme.spacing.lg}
         >
-          {allAllies.map((allies, index) => (
-            <AllyCard
-              key={index}
-              name={allies.data.Name}
-              email={allies.data.Email}
-              specialty={allies.data.Speciality}
-              location={allies.data.Zip_Code}
-            />
-          ))}
+          {allAllies.map((allies, index) => {
+            // @TODO :: Need to add ither cloundiary image, or background image to this card, as per the designs.
+            if (index === 4)
```

Comment:
Seems this would be replacing the 4th ally, rather than injecting into the 4th spot and making the list grow by one

Reply:
That is a very good point sir. If index is 4, I still need to output the ally card, in addition to this CTA. I'll get this fixed!

Reply:
@racedale should be fixed!